app/assets/javascripts/backbone/views/editable_breakeven_table_view.js.coffee

```diff
@@ -0,0 +1,27 @@
+class MS.Views.EditableBreakevenTableView extends Backbone.View
+
+  # This view may control multiple elements at once.
+  # The view is called from the loan questionnaires view.
+
+  events:
+    'click tr [data-action="delete"]': 'removeRow'
+    'click .actions [data-action="add"]': 'addRow'
+
+  initialize: (e) ->
+    @$el.find('tbody').sortable({
+      handle: "[data-action='move']"
+    })
+
+  addRow: (e) ->
+    e.preventDefault()
+    $button = @$(e.currentTarget)
+    $table = $button.closest('table')
+    $new_row = $table.find('tr.hidden').clone()
```

Comment:
camelCase. Please also refactor the other editable table class to use camel case.